hack/virtc/README.md

```diff
@@ -1,11 +1,11 @@
 # virtc
 
-`virtc` is a simple command-line tool that serves to demonstrate typical usage of the `virt` API.
-This is example software; unlike other projects like runc, runv, or rkt, `virt` is not a full container runtime.
+`virtc` is a simple command-line tool that serves to demonstrate typical usage of the `virtcontainers` API.
+This is example software; unlike other projects like runc, runv, or rkt, `virtcontainers` is not a full container runtime.
```

Comment:
I don't think "virtcontainers" needs to be rendered in a fixed-width font as it isn't a command. Also, in the context of these sentences you are referring to *projects*, not *commands*, again lending weight to not rendering in a fixed-width font.